eventhub/Module.md

```diff
@@ -127,17 +105,17 @@ You can specify the event hub path and the event data as parameters of the send
 This operation will return a ballerina error if the operation failed.
 
 
-## Entity Management in an Azure Event Hub 
+### Entity Management in an Azure Event Hub 
 This is the simplest scenario to manage entities related to azure event hubs. You need to obtain a connection string of 
 the name space of the event hub you want to send events. 
 
-### Step 1: Import the Azure Event Hub Ballerina Library
+#### Step 1: Import the Azure Event Hub Ballerina Library
 First, import the ballerinax/azure_eventhub module into the Ballerina project.
 ```ballerina
     import ballerinax/azure_eventhub;
 ```
 
-### Step 2: Initialize the Azure Event Hub ManagementClient
+#### Step 2: Initialize the Azure Event Hub ManagementClient
```

Comment:
management client is a variable name isn't it? Should we just call it client?

Reply:
Fixed in df7c75b . Thank you.